### DISCUSSION TITLE
Contest logging

### DISCUSSION THREAD

**Discussion by Michael-AUG** [magicbug/Cloudlog#2169] [Ideas / Feature Requests]
Hi
Would it be possible to set the tab order in the contesting window so that a TAB from Callsign goes straight to Serial (R) (or whatever the contest specifies as the receive)?
Also could you set a key command to log the contact? Perhaps ENTER?
Just ideas but I think they'd be appreciated.
Many thanks.
73 Michael GM5AUG

**Comment by AndreasK79** [comment]
Hi.

I have implemented a few shortcuts. Use space to move cursor.
I described some of it here:
https://github.com/magicbug/Cloudlog/wiki/Contesting

Andreas.

**Comment by Michael-AUG** [comment]
That’s great. Many thanks Andreas. Sri for missing that. Michael 73 de Mike GM5AUGSent from my iPhoneOn 27 May 2023, at 12:09 pm, Andreas Kristiansen ***@***.***> wrote:﻿
Hi.
I have implemented a few shortcuts. Use space to move cursor.
I described some of it here:
https://github.com/magicbug/Cloudlog/wiki/Contesting
Andreas.

**Reply by AndreasK79** [reply]
No worries. Suggestions are always welcome. Though it is a very basic contest logging, it works well enough for many people.
